static/structure.trombinoscope.json

```diff
@@ -0,0 +1,27 @@
+{
+  "trombinoscope": {
+    "id": "BDE",
+    "title": "Rejoins nous",
+    "membres":[
```

Comment:
The property name `membres` diverge from the [Trombinoscope](https://github.com/BDE-Polytech-MTP/website-front/blob/main/components/Trombinoscope.vue) component merged in `main`. Please use English to standardize the code and rename to `members` instead of `membres`.